app/Controllers/MediaController.php

```diff
@@ -41,10 +41,6 @@ public function show(Request $request, Response $response, string $userCode, str
 
         $filesystem = $this->storage;
 
-        if (isBot($request->getHeaderLine('User-Agent'))) {
-            return $this->streamMedia($request, $response, $filesystem, $media);
-        }
-
```

Comment:
You can't simply drop this, will make it work for discord, but for other crawlers like telegram/slack etc may stop working.

Reply:
Thanks, I didn't know that. I will try to make it so it's specifically skips Discord's user agent, i have to find the name of it first though.